lib/file-template.js

```diff
@@ -1,9 +1,12 @@
 'use strict'
 
-const fileTemplate = (language, version) => (
-`language: ${language}
-${language}:
-  - ${version}
-`)
+const fileTemplate=(language, version) => {
+	var config = `language: ${language}\n${language}:\n`
+	for (let i = 0; i < version.length; i++) {
```

Comment:
Prefer `.map()` over `for()` in this case.

Further reading: [.map() vs .forEach() vs for()](https://ryanpcmcquen.org/javascript/2015/10/25/map-vs-foreach-vs-for.html)